Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,400i,700" rel="stylesheet">
+
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+</head>
+<body>
+	<div class="container-fluid">
+		<div class="row">
+			<div class="col-xs-12 header">
+			</div>
+		</div>
+	</div>
+	<div class="container main-content">
+		<div class="row">
+			<div class="col-sm-4">
+				<img class="icon" src="img/code.png">
+			</div>
+			<div class="col-sm-8">
+				<h2>About</h2>
+				<p>It can be intimidating for women to lear and ask questions when they are in an 
+				<strong>extreme minority</strong>. While open and welcoming, today's budding developer community is 
+				up to <strong>91% male</strong>. If we can empower more females with the confidence in their 
+				technological capabilities <i>we can begin to change this landscape</i>.</p>
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-sm-12">
+				<h2>Our Locations</h2>
+			</div>
+		</div>	
+		<div class="row">
+			<div class="col-sm-4">
+				<h4>USA</h4>
+				<ul class="location-list">
+					<li><a href="#">Columbus</a></li>
+					<li><a href="#">New York</a></li>
+					<li><a href="#">Austin</a></li>
+					<li><a href="#">Philly</a></li>
+				</ul>
+			</div>
+			<div class="col-sm-4">
+				<h4>World</h4>
+				<ul class="location-list">
+					<li><a href="#">Ottawa</a></li>
+					<li><a href="#">Sydney</a></li>
+				</ul>
+			</div>
+			<div class="col-sm-4">
+				<img class="icon" src="img/world.png">
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-sm-12">
+				<h2>Upcoming Classes</h2>
+				<table class="table table-hover classes-table">
+					<thead>
+						<th>Location</th>
+						<th>Topic</th>
+					</thead>
+					<tbody>
+						<tr>
+							<td>New York</td>
+							<td>JavaScript & jQuery</td>
+						</tr>
+						<tr>
+							<td>San Francisco</td>
+							<td>HTML & CSS</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-sm-12">
```

Comment:
When you have multiple items as columns that belong to the same section (like here, where each column is a photo of the gallery) just use 1 `<div class="row">` to contain all those items, unless you have a good reason to not do so.

Look what happens now at my resolution. This will be fixed if you put all the photo columns in the same row
<img width="937" alt="screen shot 2018-05-13 at 5 51 55 pm" src="https://user-images.githubusercontent.com/6724081/39973598-7f476878-56d6-11e8-84a1-9c66f4208d32.png">